laboratory-servlet-api-integration/src/main/java/com/wixpress/petri/laboratory/http/LaboratoryFilter.java

```diff
@@ -121,6 +134,26 @@ public void init(FilterConfig filterConfig) throws ServletException {
         startMetricsReporterScheduler(laboratoryTopology.getReportsScheduleTimeInMillis());
 
         FilterTypeIdResolver.useDynamicFilterClassLoading();
+        cache = setupKoboshiCache(context);
+    }
+
+    private ReadOnlyTimestampedLocalCache<ConductibleExperiments> setupKoboshiCache(final ServletContext context) {
```

Comment:
perhaps move all koboshi related setup to separate file?
also, perhaps ive missed something, but the part about setting RegistryKey as an attribute on the servletContext feels like some magic. is there something in your readme i missed?